### DISCUSSION TITLE
Recommendations for a doorbell?

### DISCUSSION THREAD

**Discussion by moskovskiy82** [blakeblackshear/frigate#6532] [General]
Pretty impressed with frigate+double take. Going to take an approach where the front gate gets unlocked with the face recognition. So i'm looking for an IP doorbell which can feed the H264 stream 24|7. But as always there should be some fail prood scenario so a digital keypad is needed. And some kind of link to HA so a press on the door bell button will generate necessary events. Any recommendations?

**Comment by NickM-27** [comment]
Having tried the amcrest ad410 and the reolink doorbell I would say right now the reolink doorbell is hands down the best. It has better night vision, better integration with HA, and as long as you use the http-flv stream it has been stable for me on the most recent firmware update (released may 7)
